### DISCUSSION TITLE
Need help with running a python script when motion/object is detected

### DISCUSSION THREAD

**Discussion by aditya-5842** [blakeblackshear/frigate#13506] [General]
Let's say, I have a camera connected to Frigate and it can save the video file when there is a motion inside the video frame. But the camera feeds are distorted and I have written my logic to undistort the video feed.

Can I run my custom Python script that takes input from Frigate when the motion has been detected?
 
I found one excellent [discussion](https://github.com/blakeblackshear/frigate/discussions/8165). It explains how to save the snapshots. I want to save the video feed without losing any frame after processing it with opencv.

**Comment by NickM-27** [comment]
Yes, using MQTT

**Reply by aditya-5842** [reply]
Okay @NickM-27 . Could you share any documentation or an example of running Python script (that takes video frame as input) using MQTT?
